patterns-use-cases/integrations/java-spring/src/main/java/dev/restate/examples/AppMain.java

```diff
@@ -0,0 +1,13 @@
+package dev.restate.examples;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class AppMain {
```

Comment:
Is `AppMain` required? I recalled this is autoinjected somehow?

Reply:
in any case, let's just follow whatever start.spring.io generates.

Reply:
This got generated by their template indeed. I am not aware of a way around this